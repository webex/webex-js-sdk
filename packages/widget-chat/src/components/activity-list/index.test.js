import React from 'react';
import renderer from 'react-test-renderer';

import ActivityList from '.';

describe(`ActivityList component`, () => {
  const activities = [{
    id: `test-123-123-123-123`,
    content: `Test Activity Content 1`
  }, {
    id: `test-456-456-456-456`,
    content: `Test Activity Content 2`
  }];

  const component = renderer.create(
    <ActivityList
      activities={activities}
    />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
