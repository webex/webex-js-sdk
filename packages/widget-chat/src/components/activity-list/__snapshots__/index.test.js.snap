exports[`ActivityList component renders properly 1`] = `
<div>
  <ul>
    <div>
      <li>
        test-123-123-123-123
         - 
      </li>
    </div>
    <div>
      <li>
        test-456-456-456-456
         - 
      </li>
    </div>
  </ul>
</div>
`;
